### PR TITLE
Improve description of branch targeting for action release management

### LINKED ISSUE #2290: Typo in Finding and customizing actions
There is a sentence that has odd wording in the file "**main/content/actions/learn-github-actions/finding-and-customizing-actions.md**"

Link to the GitHub Help Documents where this can be found:

https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/finding-and-customizing-actions#using-release-management-for-your-custom-actions

Screenshot:

image

Suggested:

"[...] that the action will always include the latest updates [...]"

### PR DESCRIPTION
**Pull request by martindes01**
### Why:

- Fix #2290
- Sentence needs rephrasing/simplification

### What's being changed:

- Clarify use of target branch
- Separate clauses for ease of reading
- Remove unnecessary conjunction
- Replace pronoun to eliminate ambiguity

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md)

### CODE REVIEW

**Comment by janiceilene** [comment]
@martindes01 Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by janiceilene** [comment]
Thanks for your patience! Our small team is working our way through reviewing all of the amazing contributions ✨

**Review by megbird — APPROVED** [review]
This is such a great rewording ⚡  thank you for clearing this up for our users!

File: content/actions/learn-github-actions/finding-and-customizing-actions.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ steps:
 
 #### Using branches
 
-Referring to a specific branch means that the action will always use the latest updates on the target branch, but can create problems if those updates include breaking changes. This example targets a branch named `@main`:
+Specifying a target branch for the action means it will always run the version currently on that branch. This approach can create problems if an update to the branch includes breaking changes. This example targets a branch named `@main`:
```
Comment (DozerKatDC10) [inline content/actions/learn-github-actions/finding-and-customizing-actions.md:83]: "---7-2-9---"

**Review by DozerKatDC10 — COMMENTED** [review]
Owner Signature "---7-2-9---"

**Pull request merged**